src/utilities.jl

```diff
@@ -37,8 +37,9 @@ Test `ℓ` with random values.
 `N` random vectors are drawn from a standard multivariate Cauchy distribution, scaled with
 `scale` (which can be a scalar or a conformable vector).
 
-Each random vector is then used as an argument in `f(ℓ, ...)`. `logdensity` and
-`logdensity_and_gradient` are  recommended for `f`.
+Each random vector is then used as an argument in `f(ℓ, ...)`. [`logdensity`](@ref),
```

Comment:
Maybe more explicit:
```suggestion
Each random vector `x` is then used as an argument in `f(ℓ, x)`. [`logdensity`](@ref),
```